### ISSUE TITLE
ADA/Accessibility: focus should be redirected to the modal window when it opens

### ISSUE THREAD

**Issue by deborahgu** [VodkaBears/Remodal#220]
ADA/Accessibility: see [the incredible accessible modal](https://github.com/gdkraus/accessible-modal-dialog#features).

* Move focus to beginning of the modal when it opens
